enclave-manager/web/src/client/enclaveManager/KurtosisClient.ts

```diff
@@ -54,6 +55,10 @@ export abstract class KurtosisClient {
     console.log("baseApplicationUrl", this.baseApplicationUrl);
   }
 
+  isRunningInCloud() {
+    return this.cloudUrl.host.toLowerCase().includes(KURTOSIS_CLOUD_HOST);
+  }
```

Comment:
I think I'd have probably just made this an abstract method, and then in the local client always return false, and in the authenticated client return true

Reply:
good idea!